homeassistant/strings.json

```
@@ -71,6 +71,7 @@
         "webhook_not_internet_accessible": "Your Home Assistant instance needs to be accessible from the internet to receive webhook messages.",
         "oauth2_error": "Received invalid token data.",
         "oauth2_missing_configuration": "The component is not configured. Please follow the documentation.",
+        "oauth2_missing_credentials": "The component requires application credentials.",
```

You forced pushed over the fix that was committed.
```suggestion
        "oauth2_missing_credentials": "The integration requires application credentials.",
```